### DISCUSSION TITLE
Intended direction?

### DISCUSSION THREAD

**Discussion by newoldtrader** [KmiQ/coinbase-advanced-python#17] [General]
What is the goal for this code base?

Is it just as example code or is it intended to be the basis of other applications?

If the latter then here are some ideas.

Implement the websocket endpoints.
Implement rate limiting on all requests to coinbase

**Comment by KmiQ** [comment]
It is definitely the latter. It's already the basis of a couple of personal applications. As the repo continues having improvements, I guess it can better serve others with their projects as well.

I agree WebSocket endpoints and rate limiting on all requests are good things to add. Let's add issues to track it.

**Reply by newoldtrader** [reply]
What is the minimum python version being targeted?
